license-header.txt

```diff
@@ -0,0 +1,9 @@
+/**
+@license @nocompile
```

Comment:
Is it ok for these to be on the same line?

Reply:
sure is!